modules/abi-pca-resources-services.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-pca-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="abi-pca-resources-services_{context}"]
+= Creating {oci-pca-no-rt} infrastructure resources and services
+
+You must create an {oci-pca-short} environment on your virtual machine (VM) or bare-metal shape. By creating this environment, you can install {product-title} and deploy a cluster on an infrastructure that supports a wide range of cloud options and strong security policies. Having prior knowledge of {oci} components can help you with understanding the concept of {oci} resources and how you can configure them to meet your organizational needs.
+
+[IMPORTANT]
+====
+To ensure compatibility with {product-title}, you must set `A` as the record type for each DNS record and name records as follows:
+
+* `api.<cluster_name>.<base_domain>`, which targets the `apiVIP` parameter of the API load balancer.
+* `api-int.<cluster_name>.<base_domain>`, which targets the `apiVIP` parameter of the API load balancer.
+* `*.apps.<cluster_name>.<base_domain>`, which targets the `ingressVIP` parameter of the Ingress load balancer.
```

Comment:
I don't think you need the periods here, as these are not sentences, per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists).
```suggestion
* `api.<cluster_name>.<base_domain>`, which targets the `apiVIP` parameter of the API load balancer
* `api-int.<cluster_name>.<base_domain>`, which targets the `apiVIP` parameter of the API load balancer
* `*.apps.<cluster_name>.<base_domain>`, which targets the `ingressVIP` parameter of the Ingress load balancer
```
But, I see that other abi-* modules use periods in this list.

Reply:
This overall page is a rough duplicate of the existing Agent procedures in the OCI section (installing on OCI and installing on Compute Cloud@Customer). I'm replicate any non-technical feedback mentioned in this PR on those docs as well, where applicable